superset/extensions/ssh.py

```diff
@@ -53,8 +53,9 @@ def create_tunnel(
         params = {
             "ssh_address_or_host": (ssh_tunnel.server_address, ssh_tunnel.server_port),
             "ssh_username": ssh_tunnel.username,
-            "remote_bind_address": (url.host, url.port),  # bind_port, bind_host
+            "remote_bind_address": (url.host, url.port),
             "local_bind_address": (self.local_bind_address,),
+            "debug_level": "ERROR",
```

Comment:
I think it's better to set this to the current log level of Superset, so that in production this is `ERROR`, but in dev this is `DEBUG` or `INFO`, eg.

You should be able to get the current log level with something like:

```python
logging.getLogger("flask_appbuilder").level
```

Reply:
Note that I think `logging.getLogger("flask_appbuilder").level` is an int, you probably need to convert to a string here. Looks like you can do something like this:

```python
log = logging.getLogger("flask_appbuilder")
str_level = logging.getLevelName(log.getEffectiveLevel())
```

Reply:
the package handles int as well